functional/testHeadlessComponents/test.properties

```diff
@@ -0,0 +1,3 @@
+github_url="https://github.com/rh-openjdk/TestHeadlessComponents.git"
```

Comment:
You do not need this file, as you are adding it into functional instead of external test group.

Reply:
okay, cool :-)